python/ray/serve/_private/deployment_state.py

```diff
@@ -1385,6 +1386,11 @@ def get_running_replica_infos(self) -> List[RunningReplicaInfo]:
             )
         ]
 
+    def get_num_running_at_target_version(self) -> int:
+        return self._replicas.count(
+            states=[ReplicaState.RUNNING], version=self._target_state.version
+        )
```

Comment:
this level of indirection hurts readability. general rule of thumb is if you are replacing a single (easily understood) line of code with a function call, it isn't worth it

Reply:
Gotcha, I've refactored it a bit. To avoid using just a single function call, I've added a few properties to `DeploymentState` to avoid reaching into private attributes. LMK if this looks better!